app/views/my_facilities/bp_controlled.csv.erb

```diff
@@ -0,0 +1,42 @@
+<% rate_name = "controlled_patients_rate" %>
+<% arr = [] %>
+<% header = ["Facilities", "Total assigned", "Total registered","6-month change"] %>
+<% (@start_period..@period).each do |period| %>
+<% header << period #MONTHLY PERCENTAGE HEADER%>
+<% header << "Ratio" #MONTHLY RATIO HEADER %>
+<%end%>
+<%arr << header%>
+<% @display_sizes.each_with_index do |size, i| #For loop for the top row/ Sites of that size %>
+  <% row = []%>
+  <% six_month_rate_change = facility_size_six_month_rate_change(@stats_by_size[size][:periods], rate_name) %>
+  <% row << "All #{Facility.localized_facility_size(size)}s" #FACILITIES%>
+  <% row << number_or_zero_with_delimiter(@stats_by_size[size][:periods][@period][:cumulative_assigned_patients]) #Assigned pt %>
+  <% row << number_or_zero_with_delimiter(@stats_by_size[size][:periods][@period][:cumulative_registrations]) #Registered pt %>
+  <% row <<  number_to_percentage(six_month_rate_change, precision: 0) #6-MONTH CHANGE %>
+  <% @stats_by_size[size][:periods].each_pair do |period, data| %>
+    <% controlled_patients_rate = data["controlled_patients_rate"] %>
+    <% row << number_to_percentage(controlled_patients_rate || 0, precision: 0) #Monthly rate change %>
+    <% row << "#{data["controlled_patients"]} / #{data["adjusted_patient_counts"]}" #Monthly ratio %>
```

Comment:
I think this could just be `data["controlled_patients_rate"] -- it looks like you are recalculating the control rate here I think.